edk2toollib/uefi/edk2/parsers/base_parser.py

```diff
@@ -24,8 +25,9 @@ def __init__(self, log="BaseParser"):
         self.ConditionalStack = []
         self.RootPath = ""
         self.PPs = []
-        self.TargetFile = None
-        self.TargetFilePath = None
+        self._Edk2PathUtil = None
+        self.TargetFile = None  # the relative path of the target file
```

Comment:
relative to what?

Reply:
I'm not super clear on what TargetFile and TargetFilePath do. They were already there.

Reply:
ok.  we need to figure it out.

Reply:
Let me see if we can drop targetfile

Reply:
Dropped